Modules/Core/Support/MailerHelper.php

```
@@ -93,4 +93,147 @@ public static function check_mail_errors(array $errors = [], $redirect = ''): vo
+    public static function email_invoice(
+        string $invoice_id,
+        $invoice_template,
+        array $from,
+        $to,
+        $subject,
+        $body,
+        $cc = null,
+        $bcc = null,
+        $attachments = null
+    ): bool {
+        $db_invoice = \Modules\Invoices\Models\Invoice::query()->where('invoice_id', $invoice_id)->first();
```

PHPMD (2.15.0)

The variable $db_invoice is not named in camelCase. (undefined)

(CamelCaseVariableName)